bsp_diff/caas/device/intel/civ/host/vm-manager/0040-Remove-user-prompts-from-host-setup.patch

```diff
@@ -0,0 +1,58 @@
+From 3aaeb88decd1c3d2a29e4045b8f793183021b492 Mon Sep 17 00:00:00 2001
+From: vdanix <vishwanathx.dani@intel.com>
+Date: Thu, 2 Feb 2023 14:52:53 +0530
+Subject: [PATCH] Remove user prompts from host setup
+
+Changes:
+- removed prompt to update QEMU
+- removed prompt to update grub
+- removed prompt for reboot
+
+Tracked-On: OAM-105716
+Signed-off-by: vdanix <vishwanathx.dani@intel.com>
+
+diff --git a/scripts/setup_host.sh b/scripts/setup_host.sh
+index 7223f17..643eb05 100755
+--- a/scripts/setup_host.sh
++++ b/scripts/setup_host.sh
+@@ -29,10 +29,6 @@ function error() {
+ function ubu_changes_require(){
+     echo "Please make sure your apt is working"
+     echo "If you run the installation first time, reboot is required"
+-    read -p "QEMU version will be replaced (it could be recovered by 'apt purge ^qemu, apt install qemu'), do you want to continue? [Y/n]" res
+-    if [ x$res = xn ]; then
+-        return 1
+-    fi
+     sudo apt install -y wget mtools ovmf dmidecode python3-usb python3-pyudev pulseaudio jq
+ 
+     # Install libs for vm-manager
+@@ -197,6 +193,12 @@ function install_vm_manager() {
+ }
+ 
+ function ubu_enable_host_gvt(){
++    if [[ ! `cat /etc/default/grub` =~ "i915.enable_gvt=1 intel_iommu=on i915.force_probe=*" ]]; then
++        sed -i "s/GRUB_CMDLINE_LINUX=\"/GRUB_CMDLINE_LINUX=\"i915.enable_gvt=1 intel_iommu=on i915.force_probe=*/g" /etc/default/grub
++        update-grub
++
++        echo -e "\nkvmgt\nvfio-iommu-type1\nvfio-mdev\n" >> /etc/initramfs-tools/modules
++        update-initramfs -u -k all
```

Comment:
This section is wrong, please check.

Reply:
@basilchew  we reslove conflict and these changes are from s/mr0/stable https://github.com/projectceladon/vm_manager/blob/celadon/s/mr0/stable/scripts/setup_host.sh#L260

Reply:
If you apply the patch, you get this:

```
function ubu_enable_host_gvt(){
    if [[ ! `cat /etc/default/grub` =~ "i915.enable_gvt=1 intel_iommu=on i915.force_probe=*" ]]; then
        sed -i "s/GRUB_CMDLINE_LINUX=\"/GRUB_CMDLINE_LINUX=\"i915.enable_gvt=1 intel_iommu=on i915.force_probe=*/g" /etc/default/grub
        update-grub

        echo -e "\nkvmgt\nvfio-iommu-type1\nvfio-mdev\n" >> /etc/initramfs-tools/modules
        update-initramfs -u -k all

    if [[ ! `cat /etc/default/grub` =~ "i915.enable_guc="(0x)?0*"7" ]] &&
       [[ ! `cat /etc/default/grub` =~ "i915.enable_gvt=1" ]]; then

        read -p "The grub entry in '/etc/default/grub' will be updated for enabling GVT-g and GVT-d, do you want to continue? [Y/n]" res
        if [ x$res = xn ]; then
            return
        fi
        if [[ ! `cat /etc/default/grub` =~ "intel_iommu=on i915.force_probe=*" ]]; then
            sed -i "s/GRUB_CMDLINE_LINUX=\"/GRUB_CMDLINE_LINUX=\"intel_iommu=on i915.force_probe=* /g" /etc/default/grub
        fi
        sed -i "s/GRUB_CMDLINE_LINUX=\"/GRUB_CMDLINE_LINUX=\"i915.enable_gvt=1 /g" /etc/default/grub
        update-grub

        echo -e "\nkvmgt\nvfio-iommu-type1\nvfio-mdev\n" >> /etc/initramfs-tools/modules
        update-initramfs -u -k all

        reboot_required=1
    fi
}

```

1) The first "if" statement is open-ended, it is missing the closing "fi"
2) If you check the logic, the second u"if" statement already covers whatever the first "if" statement is trying to do.

Therefore, please remove the first "if" statement and related code.